### PR TITLE
Added class movie with dao, service and HibernateUtil

### PR DESCRIPTION
**Pull request by vicire**

### CODE REVIEW

File: src/main/java/cinema/dao/MovieDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import cinema.model.Movie;
+import cinema.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class MovieDaoImpl implements MovieDao {
+
```
Comment (Max-Pochepets) [inline src/main/java/cinema/dao/MovieDaoImpl.java]:
```suggestion
```

File: src/main/java/cinema/dao/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @Override
+    public Movie add(Movie movie) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            Long movieId = (Long) session.save(movie);
+            transaction.commit();
+            movie.setId(movieId);
```
Comment (Max-Pochepets) [inline src/main/java/cinema/dao/MovieDaoImpl.java]: check [this](https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_01.html) one more time

File: src/main/java/cinema/service/MovieServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+import cinema.dao.MovieDao;
+import cinema.lib.Inject;
+import cinema.lib.Service;
+import cinema.model.Movie;
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+
```
Comment (Max-Pochepets) [inline src/main/java/cinema/service/MovieServiceImpl.java]:
```suggestion
```

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
make sure that all your files finish with an empty line

File: src/main/java/cinema/dao/MovieDaoImpl.java (hunk 1 above)
Comment (vicire) [reply src/main/java/cinema/dao/MovieDaoImpl.java]: Fixed

File: src/main/java/cinema/dao/MovieDaoImpl.java (hunk 2 above)
Comment (vicire) [reply src/main/java/cinema/dao/MovieDaoImpl.java]: Fixed

File: src/main/java/cinema/service/MovieServiceImpl.java (hunk 3 above)
Comment (vicire) [reply src/main/java/cinema/service/MovieServiceImpl.java]: Fixed

**Review by Max-Pochepets — APPROVED** [review]
👍

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**